### PR TITLE
LPS-148508, LPS-148592, LPS-149973 | master

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-148592
https://issues.liferay.com/browse/LPS-148508
https://issues.liferay.com/browse/LPS-149973

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBAdmin.testcase [hunk 1]
```diff
@@ -484,9 +484,7 @@ definition {
 	@description = "This is a test for LPS-120917. It checks that XSS cannot be executed from the KB template name."
 	@priority = "3"
 	test CannotStoreXSSInTemplateName {
-		KBAdmin.openKBAdmin(siteURLKey = "guest");
-
-		KBTemplate.addCP(
+		JSONKnowledgeBase.addkBTemplate(
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBAdmin.testcase]: @chaowang-123 please try to search for existing JSON macros to use first before using the ones in the UI. If there isn't one that exists feel free to create one (and/or ask for help to create one). Thanks!

Comment (chaowang-123) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBAdmin.testcase]: @austinchiang ok, got it, thanks!

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**